### ISSUE TITLE
Numern runden/teilen, Einheiten Ändern - Kwh ist in vielen Bereichen falsch.

### ISSUE THREAD

**Issue by Alex0176** [ioBroker/ioBroker.vis-2-widgets-energy#74]
Hallo,

im Widget steht überall Kwh als Einheit und das stimmt schlichtweg nicht. 

Die Photovoltaik produziert gerade Watt oder Kilowatt, aber der Moment dauert keine Stunde an bzw. bekommt man Werte zb. in Watt und würde diese gerne in KW anzeigen.

Nicht Kilowattstunden. Desweiteren bringen einen die Nachkommastellen nichts bzw. wäre ein Teiler auch wünschenswert. 

Bei der Batterie wäre nicht nur die aktuelle Leistung (rein/raus) interessant, sondern auch die SOC (Ladezustand in %) als zusätzlicher Datenpunkt. 

<img width="360" alt="image" src="https://user-images.githubusercontent.com/18588859/232238387-0cae0747-b52b-45cf-a8f8-4b66bfb33ef2.png">

**Comment by soho737** [comment]
Second that! Tolles Widget aber vor allem die Nachkommastellen bei der Rundung sollten konfigurierbar sein. Ebenso wäre ein "Cutoff"-Wert toll, um z.B. Standby des Wechselrichters mit ein paar Watt nicht als Erzeugung/Lieferung darzustellen.

**Comment by Alex0176** [comment]
Auch das invertieren von Werten wäre praktisch, denn die Animation der Batterie wird verkehrt angezeigt. Denn ein - Wert bedeutet entladen, aber die Animation sagt Laden.

**Comment by GermanBluefox** [comment]
Neue Felder hinzugefügt:
![image](https://github.com/ioBroker/ioBroker.vis-2-widgets-energy/assets/4582016/b2409d3e-bbda-459b-b1a5-eeed33684d02)

**Comment by hombach** [comment]
Das mit dem Ausblenden funktioniert so nur bedingt... Beim PV Wechselrichter ist es so gut. Für eine Batterie noch nicht perfekt, denn Wert deutlich unter Null werden damit auch ausgeblendet... Besser wäre "Ausblenden, wenn Betrag kleiner als" ..... ;)

**Comment by hombach** [comment]
Die kWh stören im Balkendiagramm auch etwas.... extra zu beschreiben oder von den Kanälen übernehmen, sofern dort alle gleich wäre schick
![image](https://github.com/ioBroker/ioBroker.vis-2-widgets-energy/assets/63002482/37986a2c-8ba2-490a-a967-293e1662bb5b)

**Comment by GermanBluefox** [comment]
> Das mit dem Ausblenden funktioniert so nur bedingt... Beim PV Wechselrichter ist es so gut. Für eine Batterie noch nicht perfekt, denn Wert deutlich unter Null werden damit auch ausgeblendet... Besser wäre "Ausblenden, wenn Betrag kleiner als" ..... ;)

Hast du versucht -10 oder so zu schreiben?
![image](https://github.com/ioBroker/ioBroker.vis-2-widgets-energy/assets/4582016/bc7443cf-01ab-4bd4-8b6e-879c6d5f1bf7)

**Comment by hombach** [comment]
ja, meinte Absolutwert .... geht ja beim CutOff um das Filtern vom Messwert nahe Null, das ist bei Batterien halt auch mal unter Null....

**Comment by AlexB1982** [comment]
Hallo. Bei mir wird der Wert der "Stromleitungsrückgabe-OID" immer noch mit kWH angezeigt. Die Stromleitung-OID wird richtig in der Einheit (wie von mir eingestellt) als W dargestellt. 
Könnte man das bitte noch anpassen?
Danke. Gruß.
![2024-03-09 19_39_21-ioBroker vis – Mozilla Firefox](https://github.com/ioBroker/ioBroker.vis-2-widgets-energy/assets/115348743/c650834b-7f71-498d-9f50-ebd399531ffb)
